### PR TITLE
Sous Chefs Adoption

### PR DESCRIPTION
**Pull request by ramereth**
- Cookstyle fixes
- Standardize files for Sous Chefs
- Update README.md
- Update metadata.rb to Sous Chefs
- Comment out broken ChefSpec
- Remove Guardfile

### CODE REVIEW

File: resources/archive.rb [hunk 1]
```diff
@@ -1,13 +1,53 @@
...
-attribute :repo, kind_of: String, name_attribute: true
-attribute :version, kind_of: String, default: 'master'
-attribute :github_user, kind_of: String
-attribute :github_token, kind_of: String
-attribute :host, kind_of: String
-attribute :owner, kind_of: String
-attribute :group, kind_of: String
-attribute :extract_to, kind_of: String, required: true
-attribute :force, kind_of: [TrueClass, FalseClass], default: false
+property :repo, kind_of: String, name_property: true
```
Comment (detjensrobert) [inline resources/archive.rb:5]: `kind_of: ` can be removed from all of these properties:
```suggestion
property :repo, String, name_property: true
```

**Review by detjensrobert — CHANGES_REQUESTED** [review]

File: resources/archive.rb (hunk 1 above)
Comment (ramereth) [reply resources/archive.rb:5]: @tas50 I feel like Cookstyle should pick this up?

Comment (damacus) [reply resources/archive.rb:5]: What does `cookstyle -A` instead of `cookstyle -a` do here?

Comment (ramereth) [reply resources/archive.rb:5]: @damacus nothing (even using the latest on the main branch of cookstyle)

**Review by detjensrobert — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [1.0.0](https://github.com/sous-chefs/github/releases/tag/1.0.0)

**Pull request merged**